BitmovinYospacePlayerExample/BitmovinYospacePlayerExample/ContentView.swift

```diff
@@ -0,0 +1,177 @@
+        playerConfig.key = playerLicenseKey
+        
+        let playbackConfig = PlaybackConfig()
+        playbackConfig.isAutoplayEnabled = true
+        playerConfig.playbackConfig = playbackConfig
+
+        // Create player instance
+        player = BitmovinYospacePlayer(
+            playerConfig: playerConfig,
+            yospaceConfig: YospaceConfig(isDebugEnabled: true),
+            integrationConfig: IntegrationConfig(enablePlayheadNormalization: true)
+        )
+
+        // Create player view configuration
+        playerViewConfig = PlayerViewConfig()
+        
+        // Create and add listener
+        playerListener = PlayerEventListener()
+        player.add(listener: playerListener)
+    }
+   
+    private func prepareDRM(config: FairplayConfig) {
+        config.prepareCertificate = { (data: Data) -> Data in
+            guard let certString = String(data: data, encoding: .utf8),
+                  let certResult = Data(base64Encoded: certString.replacingOccurrences(of: "\"", with: ""))
+            else {
+                return data
+            }
+            return certResult
+        }
+        config.prepareContentId = { (contentId: String) -> String in
+            let prepared = contentId.replacingOccurrences(of: "skd://", with: "")
+            let components: [String] = prepared.components(separatedBy: "/")
+            return components[2]
+        }
+        config.prepareMessage = { (spcData: Data, _: String) -> Data in
+            spcData
+        }
+        config.prepareLicense = { (ckcData: Data) -> Data in
+            guard let ckcString = String(data: ckcData, encoding: .utf8),
+                  let ckcResult = Data(base64Encoded: ckcString.replacingOccurrences(of: "\"", with: ""))
+            else {
+                return ckcData
+            }
+            return ckcResult
+        }
+    }
+    
+    private func loadStream(stream: Stream) {
+        guard let streamUrl = URL(string: stream.contentUrl) else { return }
+        print("Loading \(streamUrl)")
+
+        let sourceConfig = SourceConfig(url: streamUrl, type: .hls)
+
+        if let fairplayLicense = stream.fairplayLicenseUrl, let fairplayCert = stream.fairplayCertUrl {
+            let drmConfig = FairplayConfig(license: URL(string: fairplayLicense), certificateURL: URL(string: fairplayCert)!)
+
+            if let drmHeader = stream.drmHeader {
+                print("Setting DRM header")
+                drmConfig.licenseRequestHeaders = ["x-isp-token": drmHeader]
+            }
+            prepareDRM(config: drmConfig)
+            // This needs to be commented out when running simulation
+            // As simulator does not support fairplay
+            sourceConfig.drmConfig = drmConfig
+        }
+        
+        player.load(
+            sourceConfig: sourceConfig,
+            yospaceSourceConfig: stream.yospaceSourceConfig
+        )
+    }
+
+    var body: some View {
```

Comment:
File structure:
Please keep `body` the first proeprty after the `init()` and move private helper function below the body.